<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addSet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addValSet">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="delSet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>delete</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="delValSet">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="findSet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>find</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="findValSet">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="outputSet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>output</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputLabelSet">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>My Set</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableSet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Key</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="clearSet">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>480</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>My Map</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="findKeyMap">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>370</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="addValMap">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>270</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="outputLabelMap">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>420</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findMap">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>find</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableMap">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Key</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="delMap">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>310</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearMap">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>480</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addMap">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>260</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outputMap">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>410</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>output</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="delKeyMap">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>310</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="addKeyMap">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
